tensorboard/plugins/mesh/tf_mesh_dashboard/mesh-loader.js

```diff
@@ -153,6 +153,11 @@ Polymer({
         this.appendChild(this._meshViewer.getRenderer().domElement);
         this._meshViewerAttached = true;
       }
+    }).catch(function (error) {
```

Comment:
Per Google JS style, prefer arrow functions over anonymous `function`
expressions, to make it clear that there’s no dynamic `this`-binding
going on:
<https://google.github.io/styleguide/jsguide.html#features-functions-arrow-functions>